test/functional/apps/dashboard/dashboard_state.js

```diff
@@ -278,13 +278,13 @@ export default function ({ getService, getPageObjects }) {
           await retry.try(async () => {
             const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
             // The default green color that was stored with the visualization before any dashboard overrides.
-            expect(pieSliceStyle.indexOf('rgb(87, 193, 123)')).to.be.greaterThan(0);
+            expect(pieSliceStyle.indexOf('rgb(84, 179, 153)')).to.be.greaterThan(0);
```

Comment:
I've created [this](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4424) issue for removing hard coded color values from functional tests.